engine/src/main/java/org/terasology/physics/engine/PhysicsEngineManager.java

```diff
@@ -39,6 +41,6 @@
      * @return The created {@link PhysicsEngine} instance.
      */
     public static PhysicsEngine getNewPhysicsEngine(Context context) {
-        return new BulletPhysics(context.get(WorldProvider.class));
+        return new BulletPhysics();
```

Comment:
This seems like a change that might be meaningful to explain: as per the _name_ of this method it could make perfect sense to return a new physics engine, but what did adding the `WorldProvider` before achieve that isn't needed now? And if we don't need the `Context` here should that be removed as a method param?

Reply:
world used by LiquidWorldWrapper and PhysicsWorldWrapper.
is handles "static body collision" (?)
I don't known how this doing now :D
candidates for check